eslintrc.js

```diff
@@ -0,0 +1,118 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    'jest/globals': true,
+  },
+  extends: [
+    // 'eslint:recommended',
+    // 'plugin:import/recommended',
+    // 'plugin:react/recommended',
+    // 'plugin:@typescript-eslint/eslint-recommended',
+    // 'plugin:@typescript-eslint/recommended',
+    'prettier',
+    'prettier/react',
+    'prettier/@typescript-eslint',
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    project: ['tsconfig.json', 'cypress/tsconfig.json'],
+    sourceType: 'module',
+  },
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  plugins: [
+    '@typescript-eslint',
+    '@typescript-eslint/tslint',
+    'jest',
+    'react',
+    'prettier',
+    'react-hooks',
+  ],
+  rules: {
+    '@typescript-eslint/class-name-casing': 'error',
+    '@typescript-eslint/indent': 'off',
```

Comment:
I thought this one would be covered by one of the prettier configs

Reply:
Yes, I think this was turned off by https://github.com/typescript-eslint/tslint-to-eslint-config to prevent conflicts with prettier